src/styles/card.scss

```diff
@@ -0,0 +1,100 @@
+body {
+  margin: 0;
+}
+
+.card {
+  display: flexbox;
+  box-sizing: border-box;
+  width: 200px;
+  height: 408px;
```

Comment:
No need to hardcode the height in px, it's a bad practice
```suggestion
```

Reply:
So what should I use instead? I dont really understand this comment

Reply:
card elements should stretch it, because now if I want to add something to card - the layout will be broken.